### PR TITLE
samples: wifi: qt: Add SLIP configuration

### PR DESCRIPTION
**Pull request by D-Triveni**
Add changes to use SLIP as control interface using the UART as transport.

### CODE REVIEW

File: samples/wifi/wfa_qt_app/README.rst [hunk 1]
```diff
@@ -47,6 +47,19 @@ The following is an example of the CLI command:
 
    west build -b nrf7002dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG=overlay-netusb.conf
 
+To build for the nRF7002 DK with the SLIP support, use the ``nrf7002dk_nrf5340_cpuapp`` build target with the configuration overlay :file:`overlay-slip.conf` and dtc overlay :file:`nrf7002_uart_pipe.overlay`.
```
Comment (richabp) [inline samples/wifi/wfa_qt_app/README.rst:50]: Mention the expanded form of SLIP as well.
```suggestion
To build for the nRF7002 DK with the SLIP support, use the ``nrf7002dk_nrf5340_cpuapp`` build target with the configuration overlay :file:`overlay-slip.conf` and DTC overlay :file:`nrf7002_uart_pipe.overlay`.
```

**Review by richabp — CHANGES_REQUESTED** [review]
Since we are updating this Sample, its good to fix some minor issues as well:

Line 15: The sample supports the following development **kit:**
Line 32-34: Currently, the nRF7002 DK + QSPI configuration is supported.

**Comment by D-Triveni** [comment]
> Since we are updating this Sample, its good to fix some minor issues as well:
> 
> Line 15: The sample supports the following development **kit:** Line 32-34: Currently, the nRF7002 DK + QSPI configuration is supported.

Addressed.

**Review by richabp — COMMENTED** [review]
Do we need a Changelog entry since we are adding SLIP support?

Ok, so we don't need a Changelog entry as it's already in the Changelog as a new sample.

**Comment by D-Triveni** [comment]
> Do we need a Changelog entry since we are adding SLIP support?

It's needed.

**Review by richabp — APPROVED** [review]

**Review by richabp — COMMENTED** [review]
Do we have a WFA QuickTrack (WFA QT) library as mentioned in this sample?

**Comment by D-Triveni** [comment]
> Do we have a WFA QuickTrack (WFA QT) library as mentioned in this sample?

Yes, we do.

**Review by krish2718 — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Pull request merged**